src/modules/mavlink/mavlink_main.cpp

```diff
@@ -2208,6 +2202,11 @@ Mavlink::task_main(int argc, char *argv[])
 		/* main loop */
 		px4_usleep(_main_loop_delay);
 
+		// If the wait until transmit flag is on, only transmit after we've received messages
```

Comment:
I don't  think this comment is very helpful. I'd just remove it and if someone wants to understand they have to look into the `should_transmit()` method.

Reply:
Honestly I think my brain strips comments before processing.